### ISSUE TITLE
[EstimatedMSM.pcca] First eigenvector not constant

### ISSUE THREAD

**Issue by gph82** [markovmodel/PyEMMA#424]
TLDR; 
PCCA fails because 1st REV is not constant. However, tmatrix seems to be okay. Do I really have a problem or is the definition of "not constant" too strict?

Longer version:

``` python
for iMSM in good_MSMs:
    print "is_tmatrix, is_connected = ", pyemma.msm.analysis.is_tmatrix(iMSM.transition_matrix),
    print pyemma.msm.analysis.is_connected(iMSM.transition_matrix)

#Output:
#is_tmatrix, is_connected =  True True
#is_tmatrix, is_connected =  True True
```

However, try to do PCCA on this "good" (see above) matrices:

``` python
for iMSM in good_MSMs:    
    iMSM.pcca(10)
```

Whereas the first "good_MSM" does not produce any arror, for the second I get:

``` python
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-57-8df4b6d39b23> in <module>()
      3     #plt.figure()
      4     #plt.plot(iMSM.eigenvectors_right()[:,0])
----> 5     iMSM.pcca(10)

/home/mi/gph82/programs/PyEmma/pyemma/msm/models/msm.pyc in pcca(self, m)
    780         except:
    781             # didn't have a pcca object yet - compute
--> 782             self._pcca = PCCA(self.transition_matrix, m)
    783 
    784         # set metastable properties

/home/mi/gph82/programs/PyEmma/pyemma/msm/analysis/dense/pcca.pyc in __init__(self, P, m)
    526         # PCCA memberships
    527         # TODO: can be improved. pcca computes stationary distribution internally, we don't need to compute it twice.
--> 528         self._M = pcca(P, m)
    529 
    530         # stationary distribution

/home/mi/gph82/programs/PyEmma/pyemma/msm/analysis/dense/pcca.pyc in pcca(P, m)
    429         elif (m_by_component > 1):
    430             #print "submatrix: ",closed_components_Psub[i]
--> 431             chi[component, ipcca:ipcca + m_by_component] = _pcca_connected(closed_components_Psub[i], m_by_component)
    432             ipcca += m_by_component
    433         else:

/home/mi/gph82/programs/PyEmma/pyemma/msm/analysis/dense/pcca.pyc in _pcca_connected(P, n, return_rot)
    301 
    302     # create initial solution using PCCA+. This could have negative memberships
--> 303     (chi, rot_matrix) = _pcca_connected_isa(evecs, n)
    304 
    305     #print "initial chi = \n",chi

/home/mi/gph82/programs/PyEmma/pyemma/msm/analysis/dense/pcca.pyc in _pcca_connected_isa(evec, n_clusters)
     83     if not np.isclose(diffs[0], 0):
     84         raise ValueError("First eigenvector is not constant. This indicates that the transition matrix"
---> 85                          + "is not connected or the eigenvectors are incorrectly sorted. Cannot do PCCA.")
     86     if np.isclose(min(diffs[1:]), 0):
     87         raise ValueError("An Eigenvector after the first one is constant. "

ValueError: First eigenvector is not constant. This indicates that the transition matrixis not connected or the eigenvectors are incorrectly sorted. Cannot do PCCA.
```

Plots of the respective REVs of the 1st and 2nd MSM:
![snap](https://cloud.githubusercontent.com/assets/7518004/8719709/f9f55e06-2bad-11e5-8fce-de543ed830c4.png)

Okay, the one to the right is slightly less constant. Mean and sigma values of both REVs:

``` python
print good_MSMs[0].eigenvectors_right()[:,0].mean(), good_MSMs[0].eigenvectors_right()[:,0].std()
print good_MSMs[1].eigenvectors_right()[:,0].mean(), good_MSMs[1].eigenvectors_right()[:,0].std()
#0.999999999837 7.91705540233e-11
#1.00000000088 6.25453010825e-10


```

Maximally varying element:

``` python
print np.abs(good_MSMs[0].eigenvectors_right()[:,0].mean()-good_MSMs[0].eigenvectors_right()[:,0]).max()
print np.abs(good_MSMs[1].eigenvectors_right()[:,0].mean()-good_MSMs[1].eigenvectors_right()[:,0]).max()
#8.27153678884e-10
#8.18966428007e-09
```

**Comment by franknoe** [comment]
I'll do this right away.

**Comment by franknoe** [comment]
Done, please pull msmtools and pyEMMA and try again.

**Comment by gph82** [comment]
Confirmed that this does not happen anymore. Thanks a lot!
